Bottom: 63f182d432616817a9f50e4376c5fc668ac84599
Top:    1771e6305c582f69b167cde523e973588b39c682
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-08-26 17:26:20 +0100

Refresh of hid-gtk-refactor-viewport-hand

---

diff --git a/src/hid/gtk/gui.h b/src/hid/gtk/gui.h
index 546c0eb..96c2c1b 100644
--- a/src/hid/gtk/gui.h
+++ b/src/hid/gtk/gui.h
@@ -163,8 +163,8 @@ typedef struct
   gboolean has_entered;
   gboolean panning;
 
-  Coord pcb_x, pcb_y;             /* PCB coordinates of the object under the mouse pointer */
-  Coord crosshair_x, crosshair_y; /* PCB coordinates of the crosshair set by PCB's core    */
+  Coord pcb_x, pcb_y;             /* PCB coordinates of the mouse pointer */
+  Coord crosshair_x, crosshair_y; /* PCB coordinates of the crosshair set by PCB's core */
 }
 GHidPort;
